### DISCUSSION TITLE
why docker image not support cross-platform?

### DISCUSSION THREAD

**Discussion by vncloudsco** [navidrome/navidrome#2644] [Ideas]
Why doesn't the latest version support cross-platform?

![image](https://github.com/navidrome/navidrome/assets/41409442/4a0bf6a2-755d-43f7-b644-3621576e64e3)

**Comment by deluan** [comment]
Bug. It will be fixed in the next release, due tonight: https://github.com/navidrome/navidrome/issues/2621#issuecomment-1817233434
